docker-compose-prod.yml

```diff
@@ -0,0 +1,40 @@
+version: '3.7'
+
+services:
+  web:
+    container_name: web
+    build:
+      context: .
+      dockerfile: Dockerfile
+      cache_from:
+        - "${WEB_IMAGE}"
+    image: "${WEB_IMAGE}"
+    ports:
+      - '3000:3000'
+      - '5000:5000'
+    env_file: .env
+    environment:
+      - NODE_ENV=production
+  nginx:
+    build:
+      context: .
+      cache_from:
+        - "${NGINX_IMAGE}"
+    image: "${NGINX_IMAGE}"
+    ports:
+      - 82:80
+    depends_on:
+      - web
+  db:
+    image: mongo:4.1.8-xenial
+    container_name: db
+    restart: unless-stopped
+    env_file: .env
+    environment:
+      - NODE_ENV=production
+    volumes:     
+      - dbdata:/data/db
+    ports:
+      - '27017:27017'
```

Comment:
The first port here means external port, the second mean internal port.
So the db can be accessible from outside @oesukam

Reply:
@oesukam , I removed the db container